socorro/lib/migrations.py

```diff
@@ -11,17 +11,36 @@
 
 import os
 
-
-def load_stored_proc(op, filelist):
+def get_local_filepath(filename):
     """
-    Takes a list of files and the alembic op object as arguments
-    Load and run CREATE OR REPLACE function commands from files
+    Helper for finding our raw SQL files locally.
+
     Expects files to be in:
         $CWD/socorro/external/postgresql/raw_sql/procs/
     """
     app_path=os.getcwd()
+    return (app_path + '/socorro/external/postgresql/raw_sql/procs/' + filename)
```

Comment:
Probably nothing in this case... I tend to specify full paths just in case a caller changed directories. I don't think that would happen though here. I'll give it a spin.